spark-connect/client/src/test/scala-spark-master/io/delta/connect/tables/DeltaTableSuite.scala

```diff
@@ -23,10 +23,13 @@ import org.apache.spark.sql.Row
 import org.apache.spark.sql.functions.{col, lit}
 import org.apache.spark.sql.test.DeltaQueryTest
 
+// TODO(hvanhovell) All tests here are temporary disabled. While the code compiles, we need to a
+//  Spark PR to land (https://github.com/apache/spark/pull/49971), and make a couple of changes to
+//  make tests work. This will be done in a follow-up PR.
```

Comment:
The mentioned [PR](https://github.com/apache/spark/pull/49971) landed, do the tests work now for do we need another follow up?

Reply:
We need another follow-up.